econet_tankless_water_heater.yaml

```diff
@@ -94,7 +95,26 @@ sensor:
       return max((id(temp_out).state - id(temp_in).state) * id(flow_rate).state * 8.334 * 60 / 0.92 / 1000, 0.0);
     # yamllint enable rule:line-length
     update_interval: ${econet_update_interval}
+  - platform: template
+    name: "Gas Usage"
+    id: gas_usage_m3_h
+    unit_of_measurement: "m³/h"
+    accuracy_decimals: 3
```

Comment:
If there is no device_class for gas flow rate than please set an icon and state_class for this sensor.